### ISSUE TITLE
prepare data problem

### ISSUE THREAD

**Issue by liujiaheng** [JiawangBian/SC-SfMLearner-Release#1]
![image](https://user-images.githubusercontent.com/27964642/64318034-c3c14c00-cfeb-11e9-9870-6949343a1a3c.png)
At line 108, I find that the result of `set([subdir.basename()[:-2] for subdir in subdirs])` differ every time. I find that the data structure of Set in python3.6 is unordered. I want to know which python version you use or the result of this set is unordered in other ways.
Thanks.Looking forward to your reply.

**Comment by JiawangBian** [comment]
I use python3.6. Indeed, this code is copied from Clement Pinard's pytorch implementation of SfMLearner. I tried it many times, and it works well. Maybe the order is not critical. For more details on data construction, I suggest you opening issues on his repo.

**Comment by liujiaheng** [comment]
Thanks for your reply. I am very interesting in this work. I want to verify that the 'train.txt' and 'val.txt' differ when you prepare data every time. So it may result in the training set is different from other methods, like Sfmlearner and the `np.random.seed(8964).` may be useless.  And we do not use 'val.txt' in the training process. The result works well every time maybe because the 'train.txt' is similar every time. I will continue to follow your this setting.

**Comment by JiawangBian** [comment]
Thanks for your attention. Look forward to your new conclusions about the splits.
